docs/GettingStarted.md

```diff
@@ -212,42 +174,35 @@ Finally, we'll add a field for National Insurance number. In this case, the cons
     "schemaVersion": "0.18",
     "fields": [
         { "name": "firstName", "type": "string" },
-        { "name": "age", "type": "integer"  },
-        { "name": "nationalInsurance", "type": "string"  }
+        { "name": "age", "type": "integer" },
+        { "name": "nationalInsurance", "type": "string", "nullable": true }
```

Comment:
If we do introduce a `faker.nationalInsurance` or similar type, we would probably want to change this example to avoid confusion.  Not an issue at present.

Reply:
see my comment above - this guide was written before we had a proper catalogue of types, so does need a few additions.

Reply:
I have changed the example and mention custom generators but it would be good to add a little on faker when it goes in.